src/com/company/balance/Balance.java

```diff
@@ -13,7 +13,7 @@ public Balance(UUID customerId, Double balance) {
         this.balance = balance;
     }
 
-    public abstract Double addBalance(Double additionalBalance);
```

Comment:
Please revert this change not to brake the main flow

Reply:
I`ve changed my solution base on the your advice. Can you please check it and review again?